streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -900,30 +870,22 @@ public void shouldPurgeAfterPurgeInterval() {
 
         thread.setNow(mockTime.milliseconds());
         thread.maybeCommit();
-
-        verify(taskManager);
     }
 
     @Test
     public void shouldRecordCommitLatency() {
-        final Consumer<byte[], byte[]> consumer = EasyMock.createNiceMock(Consumer.class);
         final ConsumerGroupMetadata consumerGroupMetadata = mock(ConsumerGroupMetadata.class);
-        expect(consumer.groupMetadata()).andStubReturn(consumerGroupMetadata);
-        expect(consumerGroupMetadata.groupInstanceId()).andReturn(Optional.empty());
-        expect(consumer.poll(anyObject())).andStubReturn(new ConsumerRecords<>(Collections.emptyMap()));
-        final Task task = niceMock(Task.class);
-        expect(task.id()).andStubReturn(task1);
-        expect(task.inputPartitions()).andStubReturn(Collections.singleton(t1p1));
-        expect(task.committedOffsets()).andStubReturn(Collections.emptyMap());
```

Comment:
I removed a lot of these because this is the default return behaviour in Mockito